### PR TITLE
Include default tag pending

### LINKED ISSUE #60: Make default tag "pending"
Make each candidate have a "pending" tag when they are added to HiredFiredPro
Part of issue #54

### PR DESCRIPTION
**Pull request by SQ77**
closes #60

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -45,6 +45,8 @@ public AddCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
+        tagList.add(new Tag("pending"));
```
Comment (bigismols) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:48]: To standardise should we use VALID_TAG_PENDING here?

File: src/main/java/seedu/address/model/person/Person.java [hunk 2]
```diff
@@ -35,6 +35,7 @@ public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tag
         this.email = email;
         this.address = address;
         this.tags.addAll(tags);
+        this.tags.add(new Tag("pending"));
```
Comment (bigismols) [inline src/main/java/seedu/address/model/person/Person.java:38]: Same here with VALID_TAG_PENDING

**Review by bigismols — COMMENTED** [review]
Other than minor requests, LGTM!

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 3]
```diff
@@ -45,6 +45,8 @@ public AddCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
```
Comment (oolongwater) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:46]: What is the purpose of tagList?

**Review by oolongwater — COMMENTED** [review]
Would it be better if we created a new class for status, instead of repurposing tag?

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java (hunk 3 above)
Comment (SQ77) [reply src/main/java/seedu/address/logic/parser/AddCommandParser.java:47]: It is a list that contains all the tags associated with a person. Adding the 'pending' tag to this list will make it show up even if the user does not enter it when adding the person.

Comment (oolongwater) [reply src/main/java/seedu/address/logic/parser/AddCommandParser.java:47]: isnt the staus class seperate from the current tag?

**Pull request merged**